changelog.d/20250108_184343_faraz.maqsood_add_support_for_consuming_events_from_event_bus.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Add support to consume events from event bus in discovery. Explanation can be viewed here: https://github.com/openedx/event-bus-redis/blob/main/docs/tutor_installation.rst. (by @Faraz32123)
```

Comment:
should this be an improvement or fix? We are adding new deps to fix a potentially broken behavior here.